### PR TITLE
change byte[]-string conversion method to a simpler one

### PR DESCRIPTION
**Pull request by amarseillan**

### CODE REVIEW

File: src/main/java/spark/Request.java [hunk 1]
```diff
@@ -240,7 +240,7 @@ public String body() {
     private void readBody() {
 		try {
 			bodyAsBytes = IOUtils.toByteArray(servletRequest.getInputStream());
-			body = IOUtils.toString(new ByteArrayInputStream(bodyAsBytes));
+			body = new String(bodyAsBytes);
```
Comment (jkwatson) [inline src/main/java/spark/Request.java]: both of these bother me, since we're relying on the default encoding for the system.

Comment (amarseillan) [inline src/main/java/spark/Request.java]: I agree, I didn't want to change functionality just yet, just fixing random stuff

**Comment by perwendel** [comment]
I'm sorry, but what is the gain of merging this PR?

**Comment by amarseillan** [comment]
Having 1 conversion instead of 3?
If that aint reason enough then just don't, I just think it's better code
quality

**Comment by perwendel** [comment]
I missed that. Nice tone btw!

**Comment by amarseillan** [comment]
Lol my bad :hand:

**Comment by perwendel** [comment]
:+1:

**Pull request merged**